src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -29,6 +31,24 @@
   (fn []
     (utils/show-popup (i18n/label :t/transaction-moved-title) (i18n/label :t/transaction-moved-text))))
 
+(re-frame/reg-fx
+  ::show-transaction-error
```

Comment:
Please consider reusing `ui.screens.events/:show-error`

Reply:
right we don't need it anymore, thanks

Reply:
i don't like 'Error' title tbh, because transaction timeout is not an error

Reply:
You can make `title` an optional parameter and use `Error` as default. fx could be renamed `show-popup` actually.

Reply:
ok

Reply:
as you can see i'm using here timeout, so i can't use `ui.screens.events/:show-error`

Reply:
Do you think this `timeout` usage is generic enough that it could be a parameter to `show-error`?

Reply:
hm, can't say for sure, but i think for now it's needed only here

Reply:
:+1: